pkg/model/keycloak_deployment.go

```diff
@@ -235,6 +235,8 @@ func KeycloakDeployment(cr *v1alpha1.Keycloak, dbSecret *v1.Secret) *v13.Statefu
 						},
 					},
 					ServiceAccountName: getServiceAccountName(cr),
+					Tolerations:        cr.Spec.KeycloakDeploymentSpec.Tolerations,
+					NodeSelector:       cr.Spec.KeycloakDeploymentSpec.NodeSelector,
```

Comment:
Could you please make a similar change to `KeycloakDeploymentReconciled` as well? Similarly, this needs to be in [`rhsso_deployment`](https://github.com/keycloak/keycloak-operator/blob/c24c55ac3be47a379c553c1ff9c87a8e175d7e27/pkg/model/rhsso_deployment.go).